poc_iot_verifier/src/poc_report.rs

```diff
@@ -127,7 +127,7 @@ impl Report {
     {
         sqlx::query_as::<_, Self>(
             r#"
-            select  id,
+            select poc_report.id,
```

Comment:
do we need the `poc_report` prefix on any of these now that it's the only table we're returning results from and we're explicitly setting the select query `from poc_report`?
would the `poc_report.entropy_start_time` be the same as the `entropy.timestamp` such that your filter could explicitly match
```
and entropy.timestamp = poc_report.entropy_start_time` instead of `entropy.timestamp < (NOW - Interval)`?
```

Reply:
Not sure about that. @andymck might know. We did the changes just to get it up and running live.